### DISCUSSION TITLE
Quantity data is not properly formed when calling: PATCH /v3/application/shops/{shop_id}/listings/{listing_id}

### DISCUSSION THREAD

**Discussion by carlosmartinez-dev** [etsy/open-api#1139] [Q&A]
Hello, I am getting a `400 bad request` when calling API: **PATCH** `/v3/application/shops/{shop_id}/listings/{listing_id}`, and the following error: 

**"There was a problem with \\/ : Quantity data is not properly formed."**

Sample payload is something like:
```

{
    "listing_id": 123456789,
    "state": "active",
    "taxonomy_id": 222333,
    "item_weight": 4.2,
    "item_weight_unit": "oz",
    "item_width": 11,
    "item_length": 13,
    "item_height": 1,
    "item_dimensions_unit": "in",
    "production_partner_ids": [],
    "is_digital": false,
    "return_policy_id": 1112223334440,
    "type": "physical"
}

```

Basically I want to have a clue what I have wrong at listing level or inventory level that makes the API to throw that error, or maybe even the payload itself I am using.

**Comment by diegodleon** [comment]
This seems to have started happening this past week according to our logs, and it's declining apparently, did something change on Etsy side?

<img width="986" alt="image" src="https://github.com/etsy/open-api/assets/38988761/9c56e61b-12ff-4284-8cbb-571c0cdbf6dd">

**Comment by MaximDub** [comment]
I am still having this issue myself. Any updates from the Etsy team?
